app/views/ncr/work_orders/_form.html.haml

```diff
@@ -36,3 +36,21 @@
     = f.submit class: "form-button"
     - if @model_instance.persisted?
       = link_to "Discard Changes", proposal_url(@model_instance.proposal)
+
+:javascript
```

Comment:
Might make sense to put this js in a `.js` file and just require it here -- that is where I would expect to find javascript, might unintuitive for others to look for it in here, too